src/KafkaMessageMetadataStruct.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include <stdint.h>
+#include <string>
+
+struct KafkaMessageMetadataStruct {
+  std::string PayloadToReturn;
```

Comment:
I don't think "ToReturn" adds any information, I think naming the field `Payload` would be fine.